src/Workleap.DomainEventPropagation.Subscription/DomainEventGridWebhookHandler.cs

```diff
@@ -58,14 +73,35 @@ private async Task HandleDomainEventAsync(DomainEventWrapper domainEventWrapper,
         }
 
         var domainEvent = domainEventWrapper.Unwrap(domainEventType);
+        var domainEventHandlerMethod = GetHandleDomainEventAsyncMethod(domainEventHandlerType);
+
+        await ((Task)domainEventHandlerMethod.Invoke(domainEventHandler, new[] { domainEvent, cancellationToken })!).ConfigureAwait(false);
+    }
+
+    private async Task HandleOfficevibeDomainEventAsync(EventGridEvent eventGridEvent, Type domainEventType, CancellationToken cancellationToken)
+    {
+        var domainEventHandlerType = this._domainEventTypeRegistry.GetDomainEventHandlerType(domainEventType.FullName!)!;
+        var domainEventHandler = this._serviceProvider.GetService(domainEventHandlerType);
 
-        var domainEventHandlerMethod = GenericDomainEventHandlerMethodCache.GetOrAdd(domainEventHandlerType, type =>
+        if (domainEventHandler == null)
+        {
+            this._logger.EventDomainHandlerNotRegistered(eventGridEvent.EventType);
+            return;
+        }
+
+        var domainEvent = JsonSerializer.Deserialize(eventGridEvent.Data, domainEventType);
```

Comment:
In OV they use the default serialization settings from Newtonsoft, in our WrapperClass we use default web serializer options, should we use the same here?